docs/docs/apis/openidoauth/endpoints.md

```diff
@@ -407,6 +407,79 @@ Check the [Claims](claims) page if a specific claims might be returned and for d
 
 If the token is invalid or expired, an HTTP 401 will be returned.
 
+## revocation_endpoint
+
+[https://api.zitadel.ch/oauth/v2/revoke](https://api.zitadel.ch/oauth/v2/revoke)
+
+This endpoint enables clients to revoke an `acccess_token` or `refresh_token` they have been granted.
+
+:::important
+If you revoke an `access_token` only that specific token will be revoked. If you revoke a `refresh_token`,
```

Comment:
```suggestion
If you revoke an `access_token` only the specific token will be revoked. When revoking a `refresh_token`,
```

Reply:
Is it intentional that one sentence begins with "if" and the other with "when"?

Reply:
I tried to replace the twice `If` but it sounds bad. So feel free to ignore this one 😄

Reply:
no that's good then, sounds better. just was not sure if it was intended